.github/workflows/selftests.yml

```diff
@@ -37,6 +38,10 @@ jobs:
           echo "deb http://apt.llvm.org/focal/ llvm-toolchain-focal-10 main" | sudo tee -a /etc/apt/sources.list
           sudo apt-get -qq update
           sudo apt-get -qq -y install clang-10 lld-10 llvm-10
+      - name: Install latest bpftool
+        run: |
+          git clone --depth=1 --recurse-submodules https://github.com/libbpf/bpftool bpftool
+          sudo make LLVM_STRIP=llvm-strip-10 install -C bpftool/src prefix=/usr
```

Comment:
Running 'make' as root is a bad habit, even if this is done in a container. So maybe split this into `make` followed by `sudo make install`?